requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
 pip~=22.0
 git+https://github.com/PennyLaneAI/pennylane.git@master
+datetime
```

Comment:
Is this not a built in library?

Reply:
Yes, but I was getting an error related to `datetime` in the CI so I tried upgrading it. Probably one of those CI flukes, let me try to remove it now.

Reply:
Weird. Fine to keep if needed for the CI.